External/AvalonTools/CMakeLists.txt

```diff
@@ -84,8 +84,12 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
 endif()
 
 if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
-  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-parentheses -Wno-logical-op-parentheses -Wno-dangling-else -Wno-format")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wreturn-type -Wimplicit-function-declaration -Wabsolute-value -Wno-parentheses -Wno-logical-op-parentheses -Wno-dangling-else -Wno-format")
 endif()
+if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
```

Comment:
`-Wformat-overflow=` needs a level, like `-Wformat-overflow=2`. The same is true of `-Wformat=`

Reply:
details for choices
https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html